test/inductor/test_compiled_optimizers.py

```diff
@@ -82,25 +86,36 @@ class KernelCounts(NamedTuple):
 }
 
 
-def build_compiled_opt_kwarg_db():
+def build_opt_kwarg_dbs():
     compiled_opt_db = []
+    disabled_opt_db = []
     for optim_info in optim_db:
-        if optim_info.optim_cls not in KERNEL_COUNTS:
-            continue
-
         for device in ["cpu", "cuda"]:
-            for optim_inputs in optim_info.optim_inputs_func(device):
-                for foreach in [True, False]:
-                    kwargs = dict(optim_inputs.kwargs)
-                    name = (
-                        f"test_{optim_info.optim_cls.__name__.lower()}"
-                        f"{'_foreach' if foreach else ''}"
+            for optim_inputs in _get_optim_inputs_including_global_cliquey_kwargs(
+                device, None, optim_info, skip=("fused", "differentiable")
```

Comment:
Could we track that these are currently disabled in an issue? I'm more concerned about the fused over the differentiable.

Reply:
Should be able to remove the fuse skip